app/assets/javascripts/z-bootstrap-timepicker.js

```diff
@@ -0,0 +1,1177 @@
+/*!
```

Comment:
Check out the `/vendor/assets/javascripts` directory for these kinds of 3rd-party files. Then you won't have to name them `z-` to keep them separate from your code. :smile: 

[Asset Organization](http://guides.rubyonrails.org/asset_pipeline.html#asset-organization)